rancher2/structure_cluster_rke_config_cloud_provider_vsphere.go

```diff
@@ -75,10 +75,11 @@ func flattenClusterRKEConfigCloudProviderVsphereVirtualCenter(in map[string]mana
 	}
 
 	out := make([]interface{}, len(in))
+	lenP := len(p)
```

Comment:
Why add a line of code, this is only used in a single place. I would just do it in place below.

Reply:
It's just to avoid call to `len(p)` in every loop iteration.